dolphinscheduler-api/src/main/resources/application.yaml

```diff
@@ -76,6 +76,7 @@ spring:
       org.quartz.scheduler.makeSchedulerThreadDaemon: true
       org.quartz.jobStore.driverDelegateClass: org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
       org.quartz.jobStore.clusterCheckinInterval: 5000
+      org.quartz.scheduler.batchTriggerAcquisitionMaxCount: 20
```

Comment:
In fact, we don't need to change the default value in standalone server, since the quartz will not startup in standalone server.

Reply:
can be 1 as well but need to add it

Reply:
Ok, we can add `org.quartz.scheduler.batchTriggerAcquisitionMaxCount: 1` here.

Reply:
Sure

Reply:
I will keep batchTriggerAcquisitionMaxCount to make it consistent to another quartz params in standalone server.